src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -215,13 +226,15 @@ public JdbcSourceConnectorConfig(Map<String, String> props) {
     @Override
     public List<Object> validValues(String name, Map<String, Object> config) {
       String dbUrl = (String) config.get(CONNECTION_URL_CONFIG);
+      String dbUser = (String) config.get(CONNECTION_USER_CONFIG);
+      Password dbPassword = (Password) config.get(CONNECTION_PASSWORD_CONFIG);
       String schemaPattern = (String) config.get(JdbcSourceTaskConfig.SCHEMA_PATTERN_CONFIG);
       if (dbUrl == null) {
         throw new ConfigException(CONNECTION_URL_CONFIG + " cannot be null.");
       }
       Connection db;
       try {
-        db = DriverManager.getConnection(dbUrl);
+        db = DriverManager.getConnection(dbUrl, dbUser, dbPassword == null ? null : dbPassword.value());
```

Comment:
Is `null` for the `user` parameter guaranteed to be safe? Since this goes to `DriverManager` I'd want to make sure it's generally safe, but I didn't see anything in the docs suggesting its safe. Did I miss somewhere suggesting it is safe, or should we be checking that both user and password are non-null?